docker/Dockerfile

```diff
@@ -20,6 +20,7 @@ RUN Rscript -e "renv::consent(provided = TRUE); \
       rm -rf /tmp/Rtmp*
 
 # Complete installation of zellkonverter conda env
+ENV BASILISK_EXTERNAL_DIR /usr/local/renv/basilisk
```

Comment:
How does a path like this work if someone doesn't have permissions to be working in `/usr/local`, e.g. on a server?

Reply:
Ah, yes, Dockerfile! Indeed.